openapi.yml

```diff
@@ -0,0 +1,163 @@
+openapi: 3.0.0
+info:
+  version: '1.0'
+  title: Manager of Openflow pipelines
+  description: Kytos NApp to manage pipelines on the topology
+servers:
+  - url: /api/kytos/of_multi_table
+paths:
+  /v1/pipeline:
+    get:
+      summary: Get pipelines
+      description: Get a list of pipelines
+      operationId: list_pipelines
+      responses:
```

Comment:
Same comment as https://github.com/kytos-ng/of_multi_table/pull/7/files#r1168814025, but as a list instead.

Reply:
I will keep this in mind for the next PR with mongo. It will return the object on the pipeline collection.